pyfr/plugins/fluidforce.py

```diff
@@ -108,9 +140,7 @@ def __call__(self, intg):
             uupts = solns[etype][..., self._eidxs[etype, fidx]]
 
             # Interpolate to the face
-            ufpts = np.dot(m0, uupts.reshape(nupts, -1))
-            ufpts = ufpts.reshape(nfpts, self.nvars, -1)
-            ufpts = ufpts.swapaxes(0, 1)
+            ufpts = np.dot(m0, uupts.swapaxes(0, 1)).swapaxes(0, 1)
```

Comment:
I just simplified them. I added the interpolation of gradient to the face in line 161 and implemented it in the same manner.

The dimension of m0, uupts are (nfpts, nupts) and (nupts, nvars, neles).
The dimension of np.dot(...) is (nfpts, nvars, neles). Then I swap axis 0 and 1.

In line 161,the dimension of m4 is (ndim, nfpts, nupts).
The dimension of np.dot(...) is (ndims, nfpts, nvars, neles). Then I swap axis 1 and 2 and obtain (ndims, nvars, nfpts, neles).